graf3d/eve7/src/REveCaloData.cxx

```diff
@@ -45,6 +45,9 @@ void REveCaloData::CellGeom_t::Dump() const
 
 ////////////////////////////////////////////////////////////////////////////////
 
+REveCaloData::~REveCaloData() {}
+void REveCaloData::SetSelector(REveCaloDataSelector* iSelector) { fSelector.reset(iSelector); }
```

Comment:
Can you please move those two down below REveCaloData constructor, on line 118.

Reply:
No problem, I will do it now.